src/site/stages/build/plugins/read-assets-from-disk.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+const path = require('path');
+
+function readAssetsFromDisk(buildOptions) {
+  const webpackOutput = path.join(
+    __dirname,
+    '../../../../../build/',
+    buildOptions.buildtype,
+    'generated',
+  );
+  return files => {
+    if (fs.existsSync(webpackOutput)) {
+      fs.readdirSync(webpackOutput).forEach(
+        // eslint-disable-next-line no-return-assign
```

Comment:
The reason the bot hasn't jumped all over this is because no commits have been pushed since this became a PR.